docs/reference/content/upgrade-migration/main.md

```diff
@@ -7,6 +7,15 @@ title = "Upgrade Guide"
   pre = "<i class='fa fa-cog'></i>"
 +++
 
+# What's new in 3.1
+
+- Support for MongoDB version 4.0
+- Support for Transactions
+- Support for authentication with SCRAM-SHA-256
+- Support for new URL parser (opt in with `useNewUrlParser` option)
```

Comment:
If we're going to mention the `useNewUrlParser` option, could we say that it's an option that can be set in the url?

Reply:
AFAIK we cannot set `useNewUrlParser` in the URL, since we would need a url parser to parse that url option.

Reply:
Omg of course. I meant can we say it's a client option?

Reply:
changed it to "(opt in with `useNewUrlParser` option for `MongoClient`)"